contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -456,6 +509,99 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
                 );
     }
 
+    function _exerciseLosingTickets(
+        address[] memory _tickets,
+        uint _numOfTicketsToExercise
+    ) internal returns (uint numOfTicketsToExercise) {
+        ISportsAMMV2 sportsAMM = ISportsAMMV2(manager.sportsAMM());
```

Comment:
this is called for every market in the batch and its an external call, its better to pass it as a param